scripts/python/pyembed/read_tmp_ascii.py

```diff
@@ -20,21 +20,14 @@
 
 import argparse
 
-def read_tmp_ascii(filename):
-    """
-    Arguments:
-        filename (string): temporary file created by write_tmp_point.py or write_tmp_mpr.py
+try:
+    from python_embedding import pyembed_tools
+except:
+    from pyembed.python_embedding import pyembed_tools
```

Comment:
Why is there a try/except block here for the import? Shouldn't it just be imported one way only?

Reply:
The path for python (MET_BASE/python or \<MET_installation_dir\>/share/met/python should be added for the python by MET because of met.point or met.dataplane. The try and catch is a backup for standalone python test. I will remove try/catch.

Reply:
I will put try/catch block back. It fails when PYTHON_EXE_BIN is defined.
```
TEST: python_point2grid_pb2nc_TMP_user_python           - FAIL -   2.205 sec
export MET_PYTHON_EXE='${MET_TEST_MET_PYTHON_EXE}'
/d1/personal/hsoh/git/features/feature_2285_read_point_data/MET/share/met/../../bin/point2grid \
      'PYTHON_NUMPY=/d1/personal/hsoh/git/features/feature_2285_read_point_data/MET/share/met/python/examples/read_met_point_obs.py /d1/personal/hsoh/MET/test_output/feature_2285_read_point_data/pb2nc/ndas.20120409.t12z.prepbufr.tm00.nc' \
      G212 \
      /d1/personal/hsoh/MET/test_output/feature_2285_read_point_data/python/pb2nc_TMP_user_python.nc \
      -field 'name="TMP"; level="*"; valid_time="20120409_120000"; censor_thresh=[ <0 ]; censor_val=[0];' \
      -name TEMP \
      -v 1

Traceback (most recent call last):
  File "/d1/personal/hsoh/git/features/feature_2285_read_point_data/MET/share/met/python/pyembed/read_tmp_point_nc.py", line 12, in <module>
    from python_embedding import pyembed_tools
ModuleNotFoundError: No module named 'python_embedding'
```

Reply:
That is OK. This code is mostly internal to MET and for developers, I don't see much use by outside users. I was just curious why it was needed but this error helps me understand. Thank you Howard.